recipes/acado/all/patches/0002-read-template-path-from-env.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/acado/code_generation/export_templated_file.cpp b/acado/code_generation/export_templated_file.cpp
+index 76ec47a2..937a9c7b 100644
+--- a/acado/code_generation/export_templated_file.cpp
++++ b/acado/code_generation/export_templated_file.cpp
+@@ -34,6 +34,8 @@
+ #include <acado/code_generation/export_templated_file.hpp>
+ #include <acado/code_generation/templates/templates.hpp>
+ 
++#include <cstdlib>
++
+ using namespace std;
+ 
+ BEGIN_NAMESPACE_ACADO
+@@ -66,7 +68,10 @@ returnValue ExportTemplatedFile::setup(	const std::string& _templateName,
+ {
+     ExportFile::setup( _fileName, _commonHeaderName, _realString, _intString, _precision, _commentString );
+     
+-	folders = TEMPLATE_PATHS;
++	if(const char* template_paths = std::getenv("ACADO_TEMPLATE_PATHS"))
```

Comment:
Are you going to try to upstream this patch?

Reply:
the project is dead unfortunately.

Reply:
It's the packaging into Conan that breaks the project structure, not really a bug as much as a feature from what I see